guides/micronaut-cache-hazelcast/micronaut-cache-hazelcast.adoc

```diff
@@ -0,0 +1,171 @@
+include::{commondir}/common-header.adoc[]
+
+In this guide, you will use Micronaut caching annotations to speed up your application.
+
+You are going to use a cache powered by https://hazelcast.com[Hazelcast].
+
+____
+Hazelcast is a streaming and memory-first application platform for fast, stateful, data-intensive workloads on-premises, at the edge or as fully managed cloud service.
+____
+
+include::{commondir}/common-requirements.adoc[]
+
+include::{commondir}/common-completesolution.adoc[]
+
+include::{commondir}/common-cli-or-launch.adoc[]
+
+[source,bash]
+----
+mn create-app \
+   example.micronaut.micronautguide \
+   --features=cache-hazelcast,testcontainers \ 
+   --build=@build@ \
+   --lang=@lang@ \
+----
+
+include::{commondir}/common-build-lang-arguments.adoc[]
+
+include::{commondir}/common-default-package.adoc[]
+
+NOTE: If you use https://launch.micronaut.io[Micronaut Launch], select "Micronaut Application" as application type and add `cache-hazelcast` and `testcontainers` features.
+
+include::{commondir}/common-annotationprocessors.adoc[]
+
+=== Configure the Application
+
+In this sample application, we cache news headlines. Add https://micronaut-projects.github.io/micronaut-cache/latest/guide/#hazelcast[Micronaut Hazelcast] dependency which adds support for cache using http://hazelcast.com[Hazelcast].
+
+dependency:micronaut-cache-hazelcast[groupId=io.micronaut.cache]
+
+Configure your caches in `application.yml`:
+
+resource:application.yml[tag=config]
+
+<1> Configure a cache called `headlines`.
+
+TIP: Check the https://micronaut-projects.github.io/micronaut-cache/latest/guide/configurationreference.html#io.micronaut.cache.caffeine.DefaultCacheConfiguration[properties (`maximum-size`, `expire-after-write` and `expire-after-access`)] to configure the size and expiration of your caches. It is important to keep the caches' size under control.
+
+=== Micronaut Cache API
+
+Imagine a service which retrieves headlines for a given month. This operation may be expensive and you may want to cache it.
```

Comment:
Depending on the exact retrieval operation (_e.g_, accessing a database or making an HTTP call), it may be expensive and you would want to cache it.

Reply:
Thanks, I have changed it.